client/index_old.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
We can only see your last 10 commits on this review so we can't comment on all the other changes at the correct places but I just put some additional ones here about the others parts not in these commits

Reply:
Functions in the final commit for the script.js are all well displayed and easily readable. You can follow through what each one does quite well

Reply:
The RegExp function looks very cool, I'm definitely going to look that one up

Reply:
Thanks for all the comments guys! We're gonna carry on tweaking the project today and work on the issues you mentioned, and hopefully add some functionality and perfect the styling!